### ISSUE TITLE
npm versions after 1.8.2 does not seem to include changes

### ISSUE THREAD

**Issue by Akstasy** [typesense/typesense-js#244]
I installed version 2.0.0-10 and expected to get all features and fixes. In particular `facet_strategy` in `SearchParams` interface.
When checking node_modules, version 2.0.0-10 is perfectly installed but does not include the added `facet_strategy`. 

I guess it can easily be explained by looking at the commit history:

```
commit 2e31446ec0daeff85a3f00d8adfa6ee2a6064d20 (HEAD, tag: v2.0.0-10)
Date:   Wed Nov 6 10:56:50 2024 -0600
    2.0.0-10

commit ba1cee2673b9e5a9cfb742c258183b6feee8097a
Date:   Wed Nov 6 10:54:25 2024 -0600
    Update dependencies

commit ccbaac870ea272cbeaa17e84a5477d387e5dbc00
Date:   Wed Nov 6 10:53:49 2024 -0600
    Build
    
commit 0d9415e92ac63174aa5b810d3ab0434fc19f4b25
Date:   Wed Nov 6 10:51:57 2024 -0600
    Merge pull request #241 from JinxedStar/fix-searchparam-config
    add missing facet_strategy option to SearchParams to catch up to v27
```

When checking out to tag `v2.0.0-10` and running `git diff 0d9415e92ac63174aa5b810d3ab0434fc19f4b25 "lib/Typesense/Documents.d.ts"`, i have this output:

```
diff --git a/lib/Typesense/Documents.d.ts b/lib/Typesense/Documents.d.ts
index 80c878a..afff519 100644
--- a/lib/Typesense/Documents.d.ts
+++ b/lib/Typesense/Documents.d.ts
@@ -45,7 +45,6 @@ export interface SearchParams {
     facet_query?: string;
     facet_query_num_typos?: number;
     facet_return_parent?: string;
-    facet_strategy?: 'exhaustive' | 'top_values' | 'automatic';
     page?: number;
     per_page?: number;
     group_by?: string | string[];
```

**Comment by tharropoulos** [comment]
`facet_strategy` is missing from the current HEAD of the master branch itself. It must have been some merge / rebase problem when merging #242. Added them back on #247

**Comment by tharropoulos** [comment]
The fix is now available in `v2.0.0-12`, you can give it a shot and see if it fixes your problem
